src/components/Dialog/Rename.tsx

```diff
@@ -36,7 +37,10 @@ function RenameDialog(props: Props) {
       <CloseButton onClick={props.onClose} />
       <DialogTitle>{props.title}</DialogTitle>
       <DialogContent>
-        <form style={isWidthMax500 ? { minWidth: 200 } : { minWidth: 300 }} onSubmit={handleSubmit}>
+        <form
```

Comment:
Simplification: The renaming dialog as it is right now won't be around for much longer anyway, so let's keep it simple and let's just say `{ minWidth: isSmallScreen ? 120 : 400 }` :)